components/style/stylist.rs

```diff
@@ -679,37 +679,19 @@ impl Stylist {
         parent: Option<&ComputedValues>,
         cascade_flags: CascadeFlags,
         font_metrics: &FontMetricsProvider,
-        rule_node: &StrongRuleNode
+        rule_node: StrongRuleNode
     ) -> Arc<ComputedValues> {
-        // NOTE(emilio): We skip calculating the proper layout parent style
```

Comment:
I see the comment in `compute_pseudo_element_style_with_inputs` already mentions this, but maybe it's to maintain the concrete example of fieldset?

Reply:
Sure thing :)